example/lib/main.dart

```diff
@@ -1,47 +1,78 @@
 // ignore_for_file: undefined_class, uri_has_not_been_generated
 import 'package:data_classes/data_classes.dart';
 
+// ignore: unused_import
+import 'models_helpers.dart';
+export 'models_helpers.dart';
+
 part 'main.g.dart';
 
 void main() {
-  final apple = Fruit(
+  final fruit = Fruit(
```

Comment:
@dndoja , can you keep the logic and the output of the previous example to keep it more representative? You can add you checks on top of that.